docs/catalog.md

```diff
@@ -0,0 +1,117 @@
+# What is the LEAP Data Catalog?
+
+The LEAP Data Catalog is a centralized repository that consolidates various LEAP datasets into a single JSON file. This JSON catalog is generated from individual YAML files, each representing a dataset. The catalog is currently in development, and a draft version is available at [LEAP Data Catalog](https://leap-data-catalog.vercel.app/).
+
+## The Schema
+
+The catalog is generated from individual YAML files. Each dataset, or feedstock, needs two files: `meta.yaml` and `catalog.yaml`. These files can be located in different locations to separate metadata curated during dataset creation (like Pangeo-Forge recipes) from catalog information that enhances the metadata for the LEAP catalog.
+
+### meta.yaml Schema
+
+The `meta.yaml` schema is borrowed from the [Pangeo-Forge](https://pangeo-forge.org/) project. The following fields are required:
+
+| Field         | Type             | Description                                  | Object Properties                                                                                                                                                 |
```

Comment:
I find the rendered type hinting a bit cluttered, but don't have strong feelings.

<img width="284" alt="image" src="https://github.com/leap-stc/leap-data-management-utils/assets/22455466/7d7fc269-3c93-41be-8c49-c611c0c53d03">